.circleci/config.yml

```diff
@@ -6,11 +6,12 @@ jobs:
 
   build-python36:
     docker:
-      - image: ubuntu:18.04
+      - image: ocrd/core
     steps:
       - run: apt-get update && apt-get install -y --no-install-recommends make git curl
       - checkout
-      - run: make deps-ubuntu deps-test deps install repo/assets
+      - run: make deps-ubuntu
```

Comment:
What about `deps-test`? Where will CI get pytest from?

Reply:
Oh wait, you made test depend on deps-test, right?

Reply:
Yes, exactly. It seemed more natural to encapsulate all these details. And the cost of running `pip` each time before the test seemed worth it...